staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -810,6 +819,13 @@ message ImageSpec {
     // If the runtime handler is unknown, the request should be rejected.
     // An empty string would select the default runtime handler.
     string runtime_handler = 19;
+    // Mount indicates that the OCI object (image or artifact) should be mounted.
+    // This can be done either as local host path or image ref (id or digest).
+    // Introduced in the OCI Volume Source KEP: https://kep.k8s.io/4639
+    bool mount = 20;
```

Comment:
How do we know the path it is mounted to in the PullImageResponse?

Reply:
What happens if the same image is requested to be mounted twice by the kubelet for use by different pods?

Reply:
> How do we know the path it is mounted to in the PullImageResponse?

The kubelet will not do any check (does it exist or is it mounted) on `PullImageResponse.mount_ref`. It will just use that value to map `host_path` as additional volume mount on container creation. 

We can add checks, but I see that out of scope of the kubelet's responsibility and more on a runtime leve.

> What happens if the same image is requested to be mounted twice by the kubelet for use by different pods?

CRI-O will re-use the existing mount path and not mount it again. Other runtimes are free to choose different approaches.

Reply:
Changed the API as discussed.